server/web/templates/proposals/submissions-list.handlebars

```diff
@@ -15,6 +15,9 @@
                             <div class="row">
                                 <div class="col-md-8">
                                     <h5 class="font-weight-bold">
+                                        {{#if history.isOriginal}}
```

Comment:
Change this to proposal.isOrginal, see my comments above in line https://github.com/hicsail/framingham-heart-study/pull/14/files#r1160430218